docs/src/inscriptions.md

```diff
@@ -106,3 +107,69 @@ A couple other endpoints that inscriptions may access are the following:
 - `/blockhash`: latest block hash.
 - `/blockhash/<HEIGHT>`: block hash at given block height.
 - `/blocktime`: UNIX time stamp of latest block.
+
+Provenance
+----------
+
+*NB: THIS SPEC IS NOT FINAL, NOT IMPLEMENTED, AND IS SUBJECT TO CHANGE AT ANY
+TIME. IT SHOULD NOT BE CONSIDERED FINAL UNTIL IT MAKE IT INTO A PUBLISHED,
+NUMBERED RELEASE OF `ord`. IF YOU CREATE INSCRIPTIONS BASED ON THIS IN-PROGRESS
+SPEC, IT WILL PROBABLY CHANGE AND YOU WILL PROBABLY GET REKT.*
+
+The holder of an inscription can create child inscriptions, trustlessly
+establishing the provenance of those children on-chain. This can be used for
+collections, with the children of a parent inscription being members of the
+same collection.
+
+Children can themselves have children, allowing for complex hierarchies. For
+example, an artist might create an inscription representing themselves, with
+sub inscriptions representing collections that they create, with the children
+of those sub inscriptions being items in those collections.
+
+### Specification
+
+To create a child inscription C with parent inscription P:
+
+- Create an inscribe transaction T as usual for C.
+- Spend the parent P in one of the inputs of T.
+- Include tag tag `3`, i.e. `OP_PUSH 3`, with the value of the serialized
+  inscription ID of P, `TXIDiINDEX`, serialized as the 32 byte binary `TXID`,
+  followed by the little-endian `INDEX`, with trailing zero bytes omitted.
```

Comment:
Using little-endian in combination with the term *trailing* creates ambiguity, since little-endian is "backwards" and *trailing* for it could be thought of as left-to-right.
It is clear that for address  `75f607b453ff61f63ed614c6476127f89687397488bbe6c49b151fd867f8ef5ci0` we should be recording `tag3` as `5ceff867d81f159bc4e6bb8874398796f8276147c614d63ef661ff53b407f675`.

But what if it is `75f607b453ff61f63ed614c6476127f89687397488bbe6c49b151fd867f8ef5ci31`?

Should it be `5ceff867d81f159bc4e6bb8874398796f8276147c614d63ef661ff53b407f6751f` or `5ceff867d81f159bc4e6bb8874398796f8276147c614d63ef661ff53b407f6750000001f`?

It will help if there are examples here.

Reply:
I included some examples with non-zero indices, check them out!